### PR TITLE
issue 192:Updating ip in place fqdn for  create and delete subscription

### PR DESCRIPTION
**Pull request by jeevan-kamkar**

### CODE REVIEW

**Review by bharath-b-hpe — APPROVED** [review]

File: svc-events/events/publishevttosubscriber_test.go [hunk 1]
```diff
@@ -233,6 +254,17 @@ func TestPublishEventsToDestiantion(t *testing.T) {
 		flag := PublishEventsToDestination(event)
 		assert.True(t, flag)
 	}
+	for _, v := range messages {
+		var event common.Events
+		event.IP = "10.24.1.9"
```
Comment (a-ajith) [inline svc-events/events/publishevttosubscriber_test.go:259]: give some random IP. Please change in other places also.

**Review by a-ajith — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

File: svc-events/events/publishevttosubscriber_test.go (hunk 1 above)
Comment (jeevan-kamkar) [reply svc-events/events/publishevttosubscriber_test.go:259]: all other ut in  event service  using same kind ip we will update  same later

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
Refactor code to lookupIP to a different functions - we have three copies of the same code in the same file !

**Review by skhan-anjum — APPROVED** [review]

**Comment by jeevan-kamkar** [comment]
> Refactor code to lookupIP to a different functions - we have three copies of the same code in the same file !

Changes are addressed

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**